src/pages/Index.vue

```diff
@@ -905,54 +949,57 @@
 	}
 
 	#footer {
-		display          : flex;
-		flex-direction   : column;
 		align-items      : center;
 		color            : white;
 		background-image : linear-gradient($colorBgFooterPrimary, $colorBgFooterSecondary);
 
-		.line {
-			display               : grid;
```

Comment:
When you remove the `display : grid`, then the `.line > div` elements are not aligned anymore with each other